src/pages/Landing/Header.jsx

```diff
@@ -7,34 +7,22 @@ import styles from './header.module.css';
 
 const Header = () => {
   return (
-    <div className={styles.header}>
-      <div>
-        <img alt="" className={styles.logo} src={logo} />
-      </div>
-      <div className={styles.menus}>
-        <div className={styles.menu}>
-          <a href="#sobrenos">Sobre nós</a>
-          </div>
-        <div className={styles.menu}>
-          <a href="#comofunciona">Como funciona</a>
-        </div>
-        <div className={styles.menu}>
-          <a href="#missao">Missão</a>
-        </div>
-        <div className={styles.menu}>
-          <a href="#contactos">Contactos</a>
-        </div>
-        <div className={styles.menu}>
-          <a href="#parcerias">Parcerias</a>
-        </div>
-        <div className={styles.menu}>
+    <header className={styles.header}>
+      <a href="" className={styles.logo}><img alt="" width="148px" height="50px" src={logo} /></a>
+      <input className={styles.menuBtn} type="checkbox" id="menuBtn" />
+      <label className={styles.menuIcon} htmlFor="menuBtn"><span className={styles.navicon}></span></label>
```

Comment:
Something is wrong with this line. It seems that the `id="menuBtn` is not being assumed by the htmlFor that you applied on the label.

Reply:
Yeah, it works and this error not appears for me.

Reply:
@dianper 

![Screenshot 2020-04-01 at 16 47 02](https://user-images.githubusercontent.com/20669868/78157854-70579500-7438-11ea-9aa3-1e9448c57481.png)

Reply:
We can use buttons instead of links.. or put the role on `.eslintrc` to ignore this temporally. Just to have a landing page responsive and after we can open tasks to fix them.